frontend/src/pages/MapPage/MapPage.test.tsx

```diff
@@ -0,0 +1,10 @@
+import {render} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import MapPage from "./MapPage";
+
+let component = null;
+
+test("component loads", async () => {
+    component = render(<MapPage />);
+    expect(component).toBeTruthy();
```

Comment:
Does this test that the component is visible? Can it be truthy even if it's not visible?

Reply:
Test has been updated now